### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by PyaniGnom** [community/community#125873] [GitHub Education]
### Hi everyone,

Hello, I applied on May 5 to confirm my academic status and attached a photo of my student ID. But now it's been more than three weeks and I haven't received any response, and if I follow the link in the support section - "Check if your request is pending", I am sent to the page "Upload proof of your academic status". So I can't even find out if my application is even in process.

Where can I go for help?

**Comment by s0shiro** [comment]
Here, I can track whether my application is approved or rejected: https://education.github.com/discount_requests/application
like this: 
![image](https://github.com/community/community/assets/114810349/804930d8-a486-49f3-b2d5-d2e02f259802)

**Reply by PyaniGnom** [reply]
This particular link redirects me right here, even though I've already sent a confirmation photo:

![Screenshot_1](https://github.com/community/community/assets/105264606/61f21aa8-456c-449d-bd4d-277eb04a73e3)

**Comment by jeancharlesexe** [comment]
You can check the status of your request, if it has been rejected, try to get a document that has hash verification [https://education.github.com/discount_requests/application](here )

**Comment by y0geshx** [comment]
Do it again and follow guide lines properly 🙂
- add your photo as profile picture 
- edit your GitHub profile readme
- add payment adress in GitHub settings before applying

**Reply by PyaniGnom** [reply]
Thank you, I'll try
